srcpkgs/python3-jsonpatch/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-jsonpatch'
+pkgname=python3-jsonpatch
+version=1.33
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-pypandoc"
```

Comment:
Is this correct? It breaks the build for almost all target except x86. `pypandoc` seems only be used in [`setup.py`](https://github.com/stefankoegl/python-json-patch/blob/d8e1a6e244728c04229d601bc9a384d9b034c603/setup.py#L42). So my guess it can be moved to `hostmakedepends` or be conditional on archs which don't have `python3-pypandoc` available.